essentials.yml

```diff
@@ -2,6 +2,7 @@
 ath:
   useLocalSnapshots: false
   athRevision: "acceptance-test-harness-1.78" # FTR 1.78 is 51a62122c5cda004db05599487abbb7069cf7b65
```

Comment:
Thanks @timja!

Should the `athRevision` be deleted or be changed to `acceptance-test-harness-1.80` @timja?

Reply:
ideally but i missed that when i updated it last time, that should be done to master